schema/swagger.yml

```diff
@@ -405,7 +405,7 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/UnexpectedError'
-    put:
+    patch:
```

Comment:
I think not necessary to change it to `PATCH` here.
Both `PUT` and `PATCH` make sense

Reply:
Yes its best practice to use the patch here. By convention PUT accept full resource to update. If you need to update few attributes then better to use PATCH.